README.md

```diff
@@ -131,6 +131,16 @@ Injecting `ui.tree`, `ui-tree-nodes`, `ui-tree-node`, `ui-tree-handle` to your h
 `ui-tree` is the root scope for a tree
 
 #### Attributes
+##### data-nodrop
+Turn on the ability to prevent dropping of nodes into this tree.
+- `false` (default): turn off
+- `true`: turn on no drop
+
+##### data-clone
```

Comment:
What do you think of using `data-clone-enabled`? A little more verbose and in line with `data-drag-enabled`.